Podcast/Episode.swift

```diff
@@ -100,5 +100,12 @@ class Episode: NSObject {
         // Check if series title is empty because some are
         return seriesTitle != "" ? "\(dateFormatter.string(from: dateCreated)) • \(length) • \(seriesTitle)" : "\(dateFormatter.string(from: dateCreated)) • \(length)"
     }
+    
+    func dateString() -> String {
```

Comment:
The other function we have returns "data - time - series", but in this case we just need "date."